.github/CODEOWNERS

```diff
@@ -76,10 +76,16 @@
 # Documentation
 /docs                                           @youknowriad @gziolo @chrisvanpatten @mkaz @ajitbohra @nosolosw @notnownikki
 
+# Styles (Unowned)
+*.scss                                          @ghost
+
 # Native (Unowned)
 *.native.js                                     @ghost
 *.android.js                                    @ghost
 *.ios.js                                        @ghost
 *.native.scss                                   @ghost
 *.android.scss                                  @ghost
 *.ios.scss                                      @ghost
+
+# Project Management
```

Comment:
I suspect it was intentional, but noting that with the order, this takes precedent over all previous wildcard matches (not that we'd expect SCSS files or Native files within the `.github` directory).

Reply:
Yes, I’m aware and I thought about it, too. Thanks for checking 👍